model/push_notification.go

```diff
@@ -32,7 +32,7 @@ const (
 
 	PUSH_SEND_PREPARE = "Prepared to send"
 	PUSH_SEND_SUCCESS = "Successful"
-	PUSH_NOT_SENT     = "Not Sent due to preferences"
+	PUSH_NOT_SENT     = "Not sent due to preferences"
```

Comment:
Unfortunately, we cannot change these constants due to compatibility reasons.

Reply:
Noted. Change undone.